### PR TITLE
[PPML] Add attestation for LLM Finetuning

### PR DESCRIPTION
**Pull request by xiangyuT**
## Description

Add attestation api serving for LLM Finetuning in TDX CoCo

### CODE REVIEW

**Comment by glorysdj** [comment]
please add a readme

File: docker/llm/finetune/lora/docker/Dockerfile [hunk 1]
```diff
@@ -52,7 +67,19 @@ RUN mkdir /ppml/data && mkdir /ppml/model && mkdir /ppml/output && \
     echo "    UserKnownHostsFile /dev/null" >> /etc/ssh/ssh_config && \
     sed -i 's/#\(StrictModes \).*/\1no/g' /etc/ssh/sshd_config
 
+RUN echo 'port=4050' | tee /etc/tdx-attest.conf && \
```
Comment (Uxito-Ada) [inline docker/llm/finetune/lora/docker/Dockerfile:70]: Pls combine this with the other into one RUN

File: docker/llm/finetune/lora/docker/bigdl-lora-finetuing-tdx-entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```
Comment (Uxito-Ada) [inline docker/llm/finetune/lora/docker/bigdl-lora-finetuing-tdx-entrypoint.sh:1]: what is the difference between two entrypoints? Can be only one entrypoint file here?

**Review by glorysdj — APPROVED** [review]
LGTM

File: docker/llm/finetune/lora/README.md [hunk 3]
```diff
@@ -53,3 +53,40 @@ cat launcher.log # display logs collected from other workers
 ```
 
 From the log, you can see whether finetuning process has been invoked successfully in all MPI worker pods, and a progress bar with finetuning speed and estimated time will be showed after some data preprocessing steps (this may take quiet a while).
+
+
+## To run in TDX-CoCo and enable Remote Attestation API
+
+You can deploy this workload in TDX CoCo and enable Remote Attestation API Serving with the referance yaml configuration `./kubernetes/bigdl-lora-finetuning-job.yaml.tdx`. The main diffences are it's need to execute the pods as root and mount TDX device, and a flask service is responsible for generating launcher's quote and collecting workers' quotes.
```
Comment (Uxito-Ada) [inline docker/llm/finetune/lora/README.md:60]: how to turn from normal mode to tdx mode when installing through helm? manually delete, move or rename yamls?

Comment (xiangyuT) [reply docker/llm/finetune/lora/README.md:60]: Need to manually delete and rename the yaml

**Review by Uxito-Ada — APPROVED** [review]
lgtm

**Pull request merged**